docs/maui/markup/extensions/bindable-object-extensions.md

```diff
@@ -46,18 +49,23 @@ The `Bind` method allows for a developer to supply the `Converter` that they wis
 ```csharp
 new Entry()
     .Bind(
-        nameof(ViewModel.RegistrationCode),
+        Entry.TextProperty,
+        nameof(RegistrationViewModel.RegistrationCode),
```

Comment:
```suggestion
        static (RegistrationViewModel vm) => vm.RegistrationCode,
```

Reply:
We can't as we don't have the converter support yet